src/components/Spawn/editVolumeModal/getFormSchema.tsx

```diff
@@ -0,0 +1,94 @@
+import { css } from "@emotion/react";
+import { add } from "date-fns";
+import { GetFormSchema } from "components/SpruceForm/types";
+import { ExpirationRow } from "../ExpirationRow";
+
+interface Props {
+  disableExpirationCheckbox: boolean;
+  noExpirationCheckboxTooltip: string;
+}
+
+export const getFormSchema = ({
+  disableExpirationCheckbox,
+  noExpirationCheckboxTooltip,
+}: Props): ReturnType<GetFormSchema> => ({
+  fields: {},
+  schema: {
+    type: "object",
+    properties: {
+      name: {
```

Comment:
Is it possible to disable the save button if the user changes the name to be empty? 

<img width="637" alt="Screenshot 2023-03-06 at 10 00 56 AM" src="https://user-images.githubusercontent.com/47064971/223147720-b1195cfb-6c06-4419-85e1-e20d89828289.png">